scripts/check-execution-queue.py

```
@@ -0,0 +1,645 @@
+            idx += 1
+            continue
+        break
+
+    if folded:
+        text = " ".join(value.strip() for value in values if value.strip())
+    else:
+        text = "\n".join(value.rstrip() for value in values).strip()
+    return text, idx
+
+
+def parse_frontmatter(path: Path) -> dict[str, object] | None:
```

Ruff (0.15.9)

[warning] Too many return statements (7 > 6)

(PLR0911)